docs/usage/sign_requests.md

```diff
@@ -32,12 +35,23 @@ files = [source_file]
 
 signer = {
     'name': 'John Doe',
-    'email': 'signer@mail.com' 
+    'email': 'signer@mail.com'
 }
 signers = [signer]
 
-parent_folder_id = '123456789'
-new_sign_request = client.create_sign_request(files, signers, parent_folder_id)
+new_sign_request = client.create_sign_request_v2(signers, files)
+print(f'(Sign Request ID: {new_sign_request.id})')
+```
+
+Example with sign template
+
+```python
+signer = {
+    'name': 'John Doe',
+    'email': 'signer@mail.com'
+}
+
+new_sign_request = client.create_sign_request_v2(signers, template_id = '12345')
```

Comment:
No spaces around `template_id = '12345'`, otherwise linter will complain :D

Reply:
Linter didn't complain about this particular line :D. But sure, I'll fix it

Reply:
Because it is in md file. If you paste it to .py file it would complain ;)